rwlib/source/data/Clump.hpp

```diff
@@ -149,6 +149,11 @@ struct Geometry {
         std::string name;
         std::string alphaName;
         TextureData::Handle texture;
+        Texture(std::string _name, std::string _alphaName, TextureData::Handle _texture)
+            : name(_name)
+            , alphaName(_alphaName)
```

Comment:
I believe those don't implicitly move - you might need to `std::move` them inside the initializer list.

Reply:
I will try to use perfect forwarding later. Having quite busy week. ;)

Reply:
Cool =) Though in this case you don't need perfect forwarding - `std::move` alone does fine and is the best approach no matter if you're moving or copying strings into this constructor.